### PR TITLE
Small UI updates

### PR DESCRIPTION
**Pull request by Diskyeth**
Small UI improvements and cleanup.

### CODE REVIEW

File: apps/next/components/providers.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { CreatePostProvider } from './create-post/context'
 
 const config = getDefaultConfig({
   appName: 'RumourCast App',
-  projectId: process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID || '',
+  projectId: process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID || 'c615d5a18982cf9eb69f78706b187884',
```
Comment (leovido) [inline apps/next/components/providers.tsx]: Delete the Project id key

File: apps/next/app/actions/create-post/page.tsx [hunk 2]
```diff
@@ -68,7 +68,7 @@ export default function CreatePostPage({
         <CircleHelp className="h-4 w-4" />
         <AlertTitle className="font-bold">Post anonymously to Farcaster</AlertTitle>
         <AlertDescription>
-          Must have <b>5,000 $RUMOUR</b> in your wallet to post. Posts are made anonymous
+          Must have <b>10,000,000 $RUMOUR</b> in your wallet to post. Posts are made anonymous
```
Comment (leovido) [inline apps/next/app/actions/create-post/page.tsx:71]: Where does this show? I can't find it in the current UI

**Review by leovido — APPROVED** [review]
All good now

File: apps/next/app/actions/create-post/page.tsx (hunk 2 above)
Comment (Diskyeth) [reply apps/next/app/actions/create-post/page.tsx:71]: Not sure, but changed it anyway just in case.

**Pull request merged**